pyproject.toml

```diff
@@ -73,6 +73,7 @@ dependencies = [
     "scikit-learn>=1.3",
     "scipy>=1.11",
     "Shapely>=2",
+    "splink>=3.9.12",
```

Comment:
I'm surprised that `duckdb` isn't showing up anywhere in the dependencies. Is that expected?